service_configuration_lib/spark_config.py

```diff
@@ -250,6 +250,35 @@ def _append_sql_shuffle_partitions_conf(spark_opts: Dict[str, str]) -> Dict[str,
     return spark_opts
 
 
+def _get_dra_configs(spark_opts: Dict[str, str]) -> Dict[str, str]:
+    if ('spark.dynamicAllocation.enabled' not in spark_opts or
+        str(spark_opts['spark.dynamicAllocation.enabled']) != 'true'):
+        return spark_opts
+
+    spark_opts['spark.dynamicAllocation.shuffleTracking.enabled'] = "true"
+    spark_opts['spark.dynamicAllocation.executorAllocationRatio'] = "0.8"
```

Comment:
Use global variables for specific set values like `0.8`.
Why are we setting as 0.8 and not 1.0?

Reply:
Changed to global variable. We are using 0.8 to prevent executor allocation overhead in case of small tasks